community-css-themes.json

```diff
@@ -358,5 +358,13 @@
         "screenshot": "screenshots/screenshot-themes-panel.png",
         "compatibleModes": ["light"],
         "branch": "main"
+    },
+    {
+        "name": "ITS Theme",
+        "author": "SlRvb",
+        "repo": "Obsidian--ITS-Theme",

```

Comment:
Should be `SlRvb/Obsidian--ITS-Theme` instead. Author is the display name (it can be different than the GitHub handle, in case you want something else). Hope that makes sense!